src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -10,14 +10,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RuntimeDependency Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
-      <TargetRuntime>win7-amd64</TargetRuntime>
+    <Dependency Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
       <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
-    </RuntimeDependency>
-    <RuntimeDependency Include="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni">
-      <TargetRuntime>win7-x86</TargetRuntime>
+    </Dependency>
+    <Dependency Include="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni">
       <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
-    </RuntimeDependency>
+    </Dependency>
```

Comment:
Compression is a split package that is part of the shared framework so it is allowed to use RuntimeDependencies/runtime.json.  RID specific packages are pre-resolved and stored in the shared framework directory.
SqlClient is not part of the shared framework and must work in a shared framework (aka portable/fat) application.  As such it cannot require a RID to resolve its package dependencies and cannot use RuntimeDependencies/runtime.json.